services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/error/SwitchWriteException.java

```diff
@@ -18,14 +18,17 @@
 import org.projectfloodlight.openflow.protocol.OFMessage;
 import org.projectfloodlight.openflow.types.DatapathId;
 
-public class OfWriteException extends AbstractException {
-    public OfWriteException(String s) {
-        super(s);
-    }
+public class SwitchWriteException extends SwitchOperationException {
+    private final transient OFMessage ofMessage;
```

Comment:
What's the purpose of ofMessage (as exception details) if we can't be sure that we get them (after serialization/deserial)?

Reply:
Because we do not expose FL exceptions anywhere, they are not subject to serialize/deserialize process. I don't see the reason to invent view for `OFMessage` object just because the exception object must be capable to be serializable.

PS It is handy to have access to object that case error response.